database/102-up-timescale/20240112003232-airdrop_leaderboard_only_show_once.sql

```diff
@@ -0,0 +1,83 @@
+-- migrate:up
+
+DROP FUNCTION airdrop_leaderboard;
+
+CREATE FUNCTION public.airdrop_leaderboard(epoch_ lootbox_epoch)
```

Comment:
shouldn't need to specify public

Reply:
You're right. This was derived from editing it against a local database. I'll get the updated version up.